src/components/OperationWizard.vue

```diff
@@ -78,16 +89,36 @@ const wizardTitle = computed(() => {
 function goForward() {
 	if (page.value == 'select') {
 		page.value = 'configure'
+		displayImages.value = true
 	}
 	else {
+		displayImages.value = false
 		let operationDefinition = {
 			'name': selectedOperation.value,
-			'input_data': selectedOperationInput.value
+			'input_data': {
+				...selectedOperationInput.value,
+				'input_files': selectedDataSessionImages.value
+			}
 		}
 		emit('addOperation', operationDefinition)
 	}
 }
 
+const calculateColumnSpan = (imageCount) => {
+	const imagesPerRow = 6
+	const columnsPerImage = Math.floor(12 / Math.min(imagesPerRow, imageCount))
+	return columnsPerImage
+}
+
+const handleThumbnailClick = (item) => {
+	const index = selectedDataSessionImages.value.findIndex(selectedImage => selectedImage.basename === item.basename)
+	if (index === -1) {
+		selectedDataSessionImages.value.push(item)
+	} else {
+		selectedDataSessionImages.value.splice(index, 1)
+	}
+}
+
```

Comment:
I think the picker code should be its own vue component. We already do something really similar in the project image selection view, and I suspect there will be other places we may want to use it in the future as well. It should be pretty self contained of show a bunch of image thumbnails, allowing to scroll through them and select them, and emit the selection...

Reply:
I agree AND this should be a separate story 😬